keyboards/nuphy/air75_v2/ansi/ansi.c

```diff
@@ -0,0 +1,276 @@
+
+        case MAC_DND:
+            if (record->event.pressed) {
+                host_system_send(0x9b);
+            } else {
+                host_system_send(0);
+            }
+            return false;
+
+        case SIDE_VAI:
+            if (record->event.pressed) {
+                side_light_contol(1);
+            }
+            return false;
+
+        case SIDE_VAD:
+            if (record->event.pressed) {
+                side_light_contol(0);
+            }
+            return false;
+
+        case SIDE_MOD:
+            if (record->event.pressed) {
+                side_mode_control(1);
+            }
+            return false;
+
+        case SIDE_HUI:
+            if (record->event.pressed) {
+                side_colour_control(1);
+            }
+            return false;
+
+        case SIDE_SPI:
+            if (record->event.pressed) {
+                side_speed_contol(1);
+            }
+            return false;
+
+        case SIDE_SPD:
+            if (record->event.pressed) {
+                side_speed_contol(0);
+            }
+            return false;
+
+        case DEV_RESET:
+            if (record->event.pressed) {
+                f_dev_reset_press = 1;
+                break_all_key();
+            } else {
+                f_dev_reset_press = 0;
+            }
+            return false;
+
+        case SLEEP_MODE:
+            if (record->event.pressed) {
+                if(kb_config.sleep_enable) kb_config.sleep_enable = false;
+                else kb_config.sleep_enable = true;
+                f_sleep_show       = 1;
+                eeconfig_update_kb_datablock(&kb_config);
+            }
+            return false;
+
+        case BAT_SHOW:
+            if (record->event.pressed) {
+                f_bat_hold = !f_bat_hold;
+            }
+            return false;
+
+        case BAT_NUM:
+            f_bat_num_show = record->event.pressed;
+            return false;
+
+        case RGB_TEST:
+            f_rgb_test_press = record->event.pressed;
+            return false;
+
+        default:
+            return true;
+    }
+    return true;
+}
+
+
+bool rgb_matrix_indicators_kb(void)
+{
+    if(!rgb_matrix_indicators_user()){
+        return false;
+    }
+    if(f_bat_num_show) {
+        num_led_show();
+    }
+
+    // fix power-on brightness is abnormal
+    rgb_matrix_set_color(RGB_MATRIX_LED_COUNT-1, 0, 0, 0);
+    return true;
+}
+
+/* qmk keyboard post init */
+void keyboard_post_init_kb(void) {
+
+    gpio_init();
+    rf_uart_init();
+    wait_ms(500);
+    rf_device_init();
+
+    break_all_key();
+    dial_sw_fast_scan();
+    londing_eeprom_data();
+    keyboard_post_init_user();
+}
+
+/* qmk housekeeping task */
+void housekeeping_task_kb(void) {
+
+    timer_pro();
+
+    uart_receive_pro();
+
+    uart_send_report_repeat();
+
+    dev_sts_sync();
+
+    long_press_key();
+
+    dial_sw_scan();
+
+    side_led_show();
+
+    sleep_handle();
+
+    housekeeping_task_user();
```

Comment:
Housekeeping is odd/special/a pain because it doesn't need the usual user callback. 
```suggestion
```

Reply:
Thank you for your suggestion, `housekeeping_task_user();` has been removed.